### ISSUE TITLE
Check iPad, it seems hierarchy incorrect, and tests doesn't work 

### ISSUE THREAD

**Issue by chiefchief** [mobile-dev-inc/maestro-docs#54]

**Comment by Fishbowler** [comment]
I think this issue wants to be in https://github.com/mobile-dev-inc/maestro rather than against the docs repo?
Might already be covered by https://github.com/mobile-dev-inc/maestro/issues/565

**Comment by bartekpacia** [comment]
Yes, this appears to be a duplicate of https://github.com/mobile-dev-inc/maestro/issues/565
